README.md

```diff
@@ -37,31 +51,88 @@ alongside the platform. To install your app, upload the manifest to the local
 app store on <http://127.0.0.1:8181> and it should then become visible in the
 UI, ready for you to install.
 
-## Known issues
+Databox has a number of platform components, divided into two parts:  Core and User components.
 
-### Get Started with the Graphical SDK
+### Core Components
 
-The graphical SDK will allow you to quickly build and test simple databox apps. To start the sdk run:
+* [Databox container manager](https://github.com/me-box/core-container-manager)
 
-	./databox-start sdk
+* [databox-arbiter](https://github.com/me-box/core-arbiter)
 
-The SDK web UI is available at http://127.0.0.1:8086
+* [databox-export-service](https://github.com/me-box/core-export-service)
 
-To stop the SDK run: 
+* [databox-store-json](https://github.com/me-box/store-json)
 
-	./databox-stop sdk
+* [databox-store-timeseries](https://github.com/me-box/store-timeseries)
 
+* [databox-app-server](https://github.com/me-box/platform-app-server) Server for storing and serving databox manifests
 
-### Develop system components and custom drivers
+### User Components
+#### Drivers
+* [driver-sensingkit](https://github.com/me-box/driver-sensingkit)
+* [driver-google-takeout](https://github.com/me-box/driver-google-takeout)
+* [driver-phillips-hue](https://github.com/me-box/driver-phillips-hue)
+* [driver-os-monitor](https://github.com/me-box/driver-os-monitor)
+* [driver-twitter](https://github.com/me-box/driver-twitter)
+* [driver-tplink-smart-plug](https://github.com/me-box/driver-tplink-smart-plug)    
+#### Apps 
+* [app-light-graph] https://github.com/me-box/app-light-graph
+* [app-twitter-sentiment] https://github.com/me-box/app-twitter-sentiment
+* [app-os-monitor] https://github.com/me-box/app-os-monitor
+                
+### Libraries for writing drivers and apps
+* [lib-node-databox](https://github.com/me-box/node-databox) nodejs lib for databox apps and drivers
+* [lib-python-databox](https://github.com/me-box/lib-python-databox) python lib for databox apps and drivers
+* [lib-go-databox](https://github.com/me-box/lib-go-databox) go lib for databox apps and drivers
```

Comment:
Each of the above could do with a bit of context (links to the language home pages?)
I'd also add a line after the header saying what exactly these libs provide, then simply list the different language variants of them. (After all, they should all be converging to the same API.)